OneCommunityGlobal/HighestGoodNetworkApp
# Tianjue fix red bell broken

# Description
Fix PROJECTS/TASKS/WBS COMPONENT
(PRIORITY LOW) Kaixiang/Jae: On team member tasks tab, when you click red bell icon, some task will break the page, some of them opened modal but no content, some of them did work well.

## Related Problem:
<img width="698" alt="Screen Shot 2023-03-08 at 4 12 29 PM" src="https://user-images.githubusercontent.com/70128933/223862947-f8c63c50-5aa0-4d31-a4ea-b678c05faadd.png">
The date format has been modified on the "Task Info Changes" modal.

## Mainly changes explained:
Add one condition to make sure when the old task is empty will not break the page.
Prevent reload the page before the `taskNotification` has been created.

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. login in as any role
4. go to dashboard→ Tasks→ Click on those red bells to see whether they all work properly and whether the "Task Info Changes" modal shows info correctly.

## Note:
If you do not see red bells, login in as admin, go to dashboard→ Other Links→ Projects→ AAA USE THIS PROJECT→ WBS→ Jae's WBS - USE THIS, add and edit tasks and assigned them to anyone to test.
